source/production/F0.Minesweeper.Logic/Mineplacer/FirstEmptyMineplacer.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using F0.Minesweeper.Logic.Abstractions;
+
+namespace F0.Minesweeper.Logic.Mineplacer
+{
+	internal class FirstEmptyMineplacer : IMineplacer
+	{
+		IEnumerable<ILocation> IMineplacer.PlaceMines(IEnumerable<ILocation> possibleLocations, uint mineCount, ILocation clickedLocation)
+			=> RemoveLocationsAroundClickedLocation(possibleLocations.ToList(), clickedLocation)
+				.OrderBy(_ => Guid.NewGuid())
```

Comment:
Interesting way of retrieving a specified amount of random items from a collection!
I'm not sure though if `GUID` is the right choice here, since it reprecents something _unique_, not _random_.
Although I like the `Take`, `OrderBy` would execute _deferred_ but evaluate _eagerly_, so we would still have to order the entire collection, regardless of how many we extract - I made this mistake at some coding challenge ;-(

Reply:
Opened #23